Wrappers/Python/cil/optimisation/algorithms/FISTA.py

```diff
@@ -220,6 +253,11 @@ class FISTA(ISTA):
     step_size : positive :obj:`float`, default = None
                 Step size for the gradient step of FISTA.
                 The default :code:`step_size` is :math:`\frac{1}{L}` or 1 if `f=None`.
+    step_size_rule: class with a `get_step_size` method or a function that takes an initialised CIL function as an argument and outputs a step size, default is None
+            This could be a custom `step_size_rule` or one provided in :meth:`~cil.optimisation.utilities.StepSizeMethods`. If None is passed  then the algorithm will use either `ConstantStepSize` or `ArmijioStepSize` depending on if a `step_size` is provided. 
```

Comment:
```suggestion
            This could be a custom `step_size_rule` or one provided in :meth:`~cil.optimisation.utilities.StepSizeMethods`. If None is passed  then the algorithm will use either `ConstantStepSize` or `ArmijoStepSize` depending on if a `step_size` is provided. 
```